### DISCUSSION TITLE
"Unavailable" items won't go away after attempting to delete

### DISCUSSION THREAD

**Discussion by amlloyd** [gotson/komga#1297] [Q&A]
I moved some files and directories around and now I have a couple "Unavailable" ones that won't delete. The logs say "Cannot delete series folder, path does not exist: `/comics/[path in question]`. I added the folder back, attempted to delete again which DID successfully delete the directory but the Unavailable item is still showing the the library and attempting to delete again throws the same error.

**Comment by gotson** [comment]
https://komga.org/docs/guides/trash/
